stats/stats.go

```diff
@@ -99,8 +86,32 @@ func StartStatsServerWithPort(startupOpts StartupOptions, addhandlers ...func(mu
 
 		// add the index
 		mux.HandleFunc("/", Index)
-		http.ListenAndServe(fmt.Sprintf(":%d", startupOpts.Port), mux)
+
+		server = &http.Server{
+			Addr:    fmt.Sprintf(":%d", startupOpts.Port),
+			Handler: mux,
+		}
+		_ = server.ListenAndServe()
```

Comment:
unhandled error

Reply:
Good call! It wasn't before and I lazily (incorrectly) just maintained. Added in a log line that should make things clearer